src/cfnlint/customRules/Operators.py

```diff
@@ -9,28 +9,91 @@
 def equalsOp(template, rule, propertyList):
     """ Process EQUALS operators """
     matches = []
+    for prop in propertyList:
+        actualValue = getProperty(prop, rule)
+        if actualValue.strip().lower() != str(rule.value).strip().lower():
+            matches.append(addMatches(template, rule, actualValue, prop, 'Must equal check failed'))
+    return matches
+
+
+def notEqualsOp(template, rule, propertyList):
+    """ Process NOT_EQUALS operators """
+    matches = []
+    for prop in propertyList:
+        actualValue = getProperty(prop, rule)
+        if actualValue.strip().lower() == str(rule.value).strip().lower():
+            matches.append(addMatches(template, rule, actualValue, prop, 'Must not equal check failed'))
+    return matches
+
 
+def greaterOp(template, rule, propertyList):
+    """ Process >= operators """
+    matches = []
     for prop in propertyList:
         actualValue = getProperty(prop, rule)
-        if actualValue.strip() != str(rule.value).strip():
-            path = prop['Path']
-            path = path + rule.prop.split('.')
-            linenumbers = template.get_location_yaml(template.template, path)
-            if linenumbers:
-                matches.append(Match(
-                    linenumbers[0] + 1, linenumbers[1] + 1,
-                    linenumbers[2] + 1, linenumbers[3] + 1,
-                    template.filename, CustomRule('E9201'),
-                    'Not Equal as ' + actualValue + ' does not equal ' + rule.value, None))
-            else:
-                matches.append(Match(
-                    1, 1,
-                    1, 1,
-                    template.filename, CustomRule('E9201'),
-                    'Not Equal as ' + actualValue + ' does not equal ' + rule.value, None))
+        if actualValue.strip().isnumeric() and str(rule.value).strip().isnumeric():
+            if actualValue.strip().lower() < str(rule.value).strip().lower():
+                matches.append(addMatches(template, rule, actualValue, prop, 'Greater than check failed'))
+        else:
+            matches.append(addMatches(template, rule, actualValue, prop, 'Given values are not numeric'))
     return matches
 
 
+def lessOp(template, rule, propertyList):
+    """ Process <= operators """
+    matches = []
+    for prop in propertyList:
+        actualValue = getProperty(prop, rule)
+        if actualValue.strip().isnumeric() and str(rule.value).strip().isnumeric():
+            if int(actualValue.strip()) > int(str(rule.value).strip()):
```

Comment:
So for `lessOp`, you are comparing `actualValue` against `rule.value`. But for `InSetOp` you are comparing against `rule.ruleSet`... even though they are both binary operators mathematically speaking. Can we not have just one variable for both? `rule.operand`?

Reply:
Had a Java moment there. I can absolutely use one variable "value" to represent all cases including the sets (with potential lists) and strings.